### PR TITLE
Update to v1.60.0

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xylar** [comment]
This one needs https://github.com/conda-forge/radical.utils-feedstock/pull/93 and is needed by https://github.com/conda-forge/radical.pilot-feedstock/pull/121

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,3 +44,4 @@ extra:
   recipe-maintainers:
     - andre-merzky
     - mtitov
+    - xylar
```
Comment (xylar) [inline recipe/meta.yaml:50]: I hope you don't mind another maintainer.

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by andre-merzky** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
Thanks @andre-merzky! We're going to have to wait a bit until the packages from https://github.com/conda-forge/radical.utils-feedstock/pull/93 are available.

**Comment by xylar** [comment]
@conda-forge-admin, please restart ci

**Comment by xylar** [comment]
It looks like maybe I was wrong about `radical.utils` being `noarch: python`.  It looks like weird things are happening in `win`.  I'll try to fix that over there.

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@andre-merzky, sorry for the confusion!  It should have been very obvious that we skip windows.  Somehow I missed it.

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
Travis CI is super slow, so let's see how things go with cross-compiling on Azure.

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@andre-merzky, @mtitov, this is finally ready to review (and hopefully merge).

**Comment by andre-merzky** [comment]
Hey @xylar - thanks for digging through this!

**Pull request merged**